subsys/nrf_security/src/drivers/cracen/sicrypto/src/waitqueue.h

```diff
@@ -32,6 +50,8 @@ static inline void si_wq_wake_next(struct sitask *t)
 	struct siwq *w = t->wq;
 
 	while ((w != NULL) && (t->statuscode != SX_ERR_HW_PROCESSING)) {
+		validate_function_pointer(w->run);
```

Comment:
This does not do anything - asserts are not included in release builds and calll on line 56 will anyways trap because MPU will trap on XN bit

Reply:
My mistake, good catch.

Should have tested this.